app/uk/gov/hmrc/economiccrimelevyregistration/config/AppConfig.scala

```diff
@@ -93,7 +93,7 @@ class AppConfig @Inject() (configuration: Configuration, servicesConfig: Service
 
   val privateBetaAccessCodes: Seq[String] = configuration.get[Seq[String]]("features.privateBeta.accessCodes")
   val privateBetaContactEmail: String     = configuration.get[String]("features.privateBeta.contactEmail")
-  val privateBetaEnabled: Boolean         = configuration.get[Boolean]("features.privateBeta.enabled")
+  val privateBetaEnabled: Boolean         = false // configuration.get[Boolean]("features.privateBeta.enabled")
```

Comment:
i think you've left this in here by mistake

Reply:
Correct. Npw cprrected.